benchmarks/linear_algebra/blas/level1/copy/copy.cpp

```diff
@@ -0,0 +1,56 @@
+/**
+  copy : copies vector to another vector
+
+
+  for (int i = 0; i < N; i++)
+      output[i] = A[i];
+*/
+
+#include <tiramisu/tiramisu.h>
+
+#define NN 10
+
+using namespace tiramisu;
+
+int main(int argc, char **argv)
+{
+    tiramisu::init("copy");
+
+    // -------------------------------------------------------
+    // Layer I
+    // -------------------------------------------------------
+
+    // Declare the constant N.
+    constant N("N", NN);
+
+    // Declare iterator variable.
+    var i("i", 0, N);
+
+    // Declare the input.
+    input A("A", {i}, p_uint8);
+
+    // Declare the output computation.
+    computation output("output", {i}, A(i));
+
```

Comment:
Thanks! The code looks good to me, but I have one comment. The code does not respect the Tiramisu coding style. For example, the code should not have a new line when not necessary.

Reply:
done.